### DISCUSSION TITLE
Shared element transitions?

### DISCUSSION THREAD

**Discussion by redbar0n** [tamagui/tamagui#104] [Q&A]
Figured I’d post here instead of on the Discord chat, so it’s discoverable by others with the same question:

Wondering what the story is around shared element transitions? 

Is it straightforward to use something like react-native-shared-element with Tamagui?

https://youtu.be/tC7egqnpYSY

How would it work on web, if so? Would it automatically use the new shared element transitions web API?

https://developer.chrome.com/blog/shared-element-transitions-for-spas/

**Comment by redbar0n** [comment]
Related: [FluidTransitions](https://github.com/fram-x/FluidTransitions) - "The library implements a new navigator component called FluidNavigator with the same interface and routing configuration as the StackNavigator. The library has a component called Transition which can be used to build different types of transitions that will automatically be run when navigating between screens using the regular navigation actions."

**Comment by redbar0n** [comment]
Shared Element Transitions in Greensock vs. Framer Motion vs. Shared Element Transition API: https://twitter.com/mattgperry/status/1580937285280747521?s=20&t=kMbKE8AVRPqew5YWAgPr-A I wonder how Tamagui would fare in that test..?

**Comment by redbar0n** [comment]
"We got agreement in the CSS Working Group to move the Shared Element Transitions spec to First Public Working Draft. Still lots to do, but it's moving forward nicely. https://developer.chrome.com/blog/shared-element-transitions-for-spas/"
-- Jake Archibald, 4:41 PM · Sep 16, 2022, [tweet](https://twitter.com/jaffathecake/status/1570784912856485888?s=20&t=u8EQcQbbn33-guJSc0d67A)

**Comment by dohomi** [comment]
Interested in this as well.. currently I use fullscreen dialogs which is just a workaround but fills the purpose but its not a nice UX

**Comment by jyaconelli** [comment]
stumbled across an example of using Reanimated shared transition with Tamagui https://youtu.be/r_Smo78tv8Y?t=7019
